canonical-web-and-design/ubuntu.com
# Move JS to make linting easier

## Done

- Restructure JS directory to make linting easier
- Delete unused files (cookie-proxy.js and rtp.js)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Run `./run` and make sure the build doesn't fail


## Issue / Card

Fixes #6880
